File: travel_redommendations/v1/get/Node SDK/travel_recommendations.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Recommended locations similar to PAR
    const response = await amadeus.referenceData.recommendedLocations.get({
      cityCodes: "PAR",
      travelerCountryCode: "FR",
    });

    console.log(response);
  } catch (error) {
    console.log(error);
  }
}

main();

File: travel_redommendations/v1/get/Python SDK/travel_recommendations.py
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    Recommends travel destinations similar to Paris for travelers in France
    '''
    response = amadeus.reference_data.recommended_locations.get(cityCodes='PAR', travelerCountryCode='FR')
    print(response.data)
except ResponseError as error:
    raise error